jenny/templates/deployment.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "jenny.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    {{- include "jenny.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "jenny.selectorLabels" . | nindent 6 }}
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "jenny.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- include "django.imagePullSecrets" . | nindent 6 }}
+      securityContext:
+        {{- toYaml .Values.securityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+            {{- toYaml .Values.django.command | nindent 12 }}
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+          livenessProbe:
+            httpGet:
+              path: /admin/login/
```

Comment:
It should be configurable. We usually use Dockerflow's `__heartbeat__` and `__lbheartbeat__` endpoints for probes.

Reply:
nothing is configured on django side, to you use https://python-dockerflow.readthedocs.io/en/main/django.html or anything else ?

Reply:
Yes, we integrate this project in all our Django projects!